core/src/main/java/org/mindtrails/controller/ActionSequenceController.java

```diff
@@ -0,0 +1,66 @@
+package org.mindtrails.controller;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.mindtrails.domain.ExportMode;
+import org.mindtrails.domain.Participant;
+import org.mindtrails.domain.Session;
+import org.mindtrails.domain.Study;
+import org.mindtrails.persistence.ReasonsForEnding;
+import org.mindtrails.persistence.ReasonsForEndingRepository;
+import org.mindtrails.service.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.View;
+import org.springframework.web.servlet.view.RedirectView;
+
+import java.security.Principal;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by Anna on 7/2/19
+ */
+
+@Controller
+@RequestMapping("/action")
+public class ActionController extends BaseController {
+
+    @Autowired
+    ActionSequenceRepository actionSequenceRepository;
```

Comment:
This is resulting in a compile time error.  There are lots of them - class not found exceptions because imports are not correctly defined at he top of the file. I think we need to get you up and running so you can actually test the code - locate compile issues, and fix them.  And really, see the code in action so you can test that data is being transferred correctly.  We should also write some automated tests so we can assert over time that the code is working properly and that I don't make changes later that break the work you are doing.  This becomes terribly important as we have more people working on the code.